main.js

```diff
@@ -187,79 +181,72 @@ function showTimer() {
 function clockFormat() {
   var totalSec = parseInt(currentActivity.seconds);
   var totalMin = parseInt(currentActivity.minutes * 60);
-  var time = totalSec + totalMin
-
+  var time = totalSec + totalMin;
   var minutes =  Math.floor(time/60);
   var seconds = time % 60;
 
-    minutes = minutes < 10 ? "0" + minutes : minutes;
-    seconds = seconds < 10 ? "0" + seconds : seconds;
-    timerCountdown.innerText = `${minutes}:${seconds}`
+  minutes = minutes < 10 ? "0" + minutes : minutes;
+  seconds = seconds < 10 ? "0" + seconds : seconds;
+  timerCountdown.innerText = `${minutes}:${seconds}`;
 }
 
 function startTimer() {
   var totalSec = parseInt(currentActivity.seconds);
   var totalMin = parseInt(currentActivity.minutes * 60);
-  var time = totalSec + totalMin
+  var time = totalSec + totalMin;
 
   var timer = setInterval(function() {
     var minutes =  Math.floor(time/60);
     var seconds = time % 60;
 
     minutes = minutes < 10 ? "0" + minutes : minutes;
     seconds = seconds < 10 ? "0" + seconds : seconds;
-    timerCountdown.innerText = `${minutes}:${seconds}`
+    timerCountdown.innerText = `${minutes}:${seconds}`;
     time--;
-
     if (time === -1) {
-        clearInterval(timer);
-        timerCountdown.innerHTML = "00:00";
-        completeTimer()
-         }
-    }, 1000);
+      clearInterval(timer);
+      timerCountdown.innerHTML = "00:00";
+      completeTimer();
+    }
+  }, 1000);
 }
 
 function completeTimer() {
   currentActivity.markComplete();
   if (timerCountdown.innerHTML === "00:00") {
     circleTimerBtn.innerText = 'COMPLETE!';
-    circleTimerBtn.classList.add('complete-circle')
+    circleTimerBtn.classList.add('complete-circle');
     logActivityBtn.classList.remove('hidden');
-
     }
 }
 
-var logPastActivities = document.querySelector('#listPastActivities')
+var logPastActivities = document.querySelector('#listPastActivities');
 var activitySection = document.querySelector('#activitySection');
-logActivityBtn.addEventListener('click', displayLoggedActivity)
-var createNewActivityBtn = document.querySelector('#createNewActivityButton')
+var createNewActivityBtn = document.querySelector('#createNewActivityButton');
 var buttonForm = document.querySelector('#buttonForm');
-
+logActivityBtn.addEventListener('click', displayLoggedActivity);
 createNewActivityBtn.addEventListener('click', returnCreateActivityView);
 
 
 function displayLoggedActivity() {
-activitySection.innerHTML = '';
-showCompletedActivityView ();
-
-retrieveStored();
-
+  activitySection.innerHTML = '';
+  showCompletedActivityView ();
+  retrieveStored();
```

Comment:
could reassign savedActivities to what is returned out of retrieveStored to display these cards

Reply:
That's helpful, thank you.